cmd/snap-exec/main.go

```diff
@@ -214,6 +216,13 @@ func execApp(snapApp, revision, command string, args []string) error {
 
 	fullCmd = append(absoluteCommandChain(app.Snap, app.CommandChain), fullCmd...)
 
+	if osutil.GetenvBool("SNAP_DEBUG_START_TIME") {
+		if runTimeStart, err := strconv.ParseInt(os.Getenv("SNAP_RUN_TIME_START"), 10, 64); err == nil {
+			d := time.Now().Sub(time.Unix(0, runTimeStart))
+			fmt.Fprintf(os.Stderr, "Seconds until exec %q: %v\n", fullCmd, d.Seconds())
```

Comment:
With command chains `fullCmd` can get quite long, pushing duration far to the right. How about this format:
```
fmt.Fprintf("command: %q\ntime until exec: %v\n", fullCmd, d)
```
BTW `time.Duration.String()` produces nice, human readable output, but I'm fine if you prefer using `d.Seconds()`.